.gitignore

```diff
@@ -28,7 +28,7 @@ bin/
 *~
 TAGS
 
-# storage 
+# storage
```

Comment:
Not a useful change, can be omitted or moved into a separate commit.

Reply:
Sorry it's auto whitespace trimming function :-( I'll take care of it